modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,88 @@
++
+* *Applications logs*: Lets the Operator collect application logs, which includes everything that is _not_ deployed in the `openshift-*` namespaces. Default: `yes`
+* *Infrastructure logs*: Lets the Operator collect logs from OpenShift Container Platform, Kubernetes, and some nodes. Default: `yes`
+* *Audit logs*: Type `yes` to let the Operator collect node logs related to security audits. By default, Red Hat stores audit logs outside the cluster through a separate mechanism that does not rely on the Cluster Logging Operator. For more information about default audit logging, see the ROSA Service Definition. Default: `no`
+
+. If you are installing the Amazon CloudWatch add-on, use the default log grouping type or change it. Valid values: `logType`, `namespaceName`, or `namespaceUUID`. Default: `logType`
+. For the Amazon CloudWatch region, use the default cluster region, leave the `Cloudwatch region` value empty.
++
+.Example output
+[source,terminal]
+----
+? Are you sure you want to install add-on 'cluster-logging-operator' on cluster '<cluster_name>'? Yes
+? Use AWS CloudWatch (optional): Yes
+? Collect Applications logs (optional): Yes
+? Collect Infrastructure logs (optional): Yes
+? Collect Audit logs (optional): No
+? CloudWatch log grouping type: logType
+? CloudWatch region (optional):
+I: Add-on 'cluster-logging-operator' is now installing. To check the status run 'rosa list addons --cluster=<cluster_name>'
+----
+
+[NOTE]
+====
+The installation can take approximately 10 minutes to complete.
+====
+
+.Verification steps
+
+. To verify the logging installation status, enter the following command:
++
+[source,terminal]
+----
+$ rosa list addons --cluster=<cluster_name>
+----
+
+. To verify which pods are deployed by `cluster-logging-operator` and their state of readiness:
+
+.. Log in to the `oc` CLI using `cluster-admin` credentials:
++
+[source,terminal]
+----
+$ oc login https://api.mycluster.abwp.s1.example.org:6443 \
+   --username cluster-admin
+   --password <password>
+----
+
+.. Enter the following command to get information about the pods for the default project. Alternatively, you can specify a different project.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-logging
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                        READY         STATUS      RESTARTS   AGE
+cluster-logging-operator-<pod_ID  >         2/2           Running     0          7m1s
+fluentd-4mnwp                               1/1           Running     0          6m3s
+fluentd-6xt25                               1/1           Running     0          6m3s
+fluentd-fqjhv                               1/1           Running     0          6m3s
+fluentd-gcvrg                               1/1           Running     0          6m3s
+fluentd-vpwrt                               1/1           Running     0          6m3s
```

Comment:
@jonquilwilliams The `cluster-logging-operator` add-on generates clusterlogging instance and clusterlogforwder instance. Can we add this information here?
The commands for them are:
oc get clusterlogging -n openshift-logging
oc get clusterlogforwarders -n openshift-logging

Reply:
Done.